calrissian/job.py

```diff
@@ -170,6 +171,47 @@ def container_workingdir(self):
         """
         return self.environment['HOME']
 
+    # Conversions from CWL ResourceRequirements https://www.commonwl.org/v1.0/CommandLineTool.html#ResourceRequirement
+    # to Kubernetes Resource requests/limits
+    # https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container
+
+    @staticmethod
+    def resource_bound(cwl_field):
+        if cwl_field.endswith('Min'):
+            return 'requests'
+        elif cwl_field.endswith('Max'):
+            return 'limits'
+        else:
+            return None
+
+    @staticmethod
+    def resource_type(cwl_field):
+        if cwl_field.startswith('cores'):
+            return 'cpu'
+        elif cwl_field.startswith('ram'):
+            return 'memory'
+        else:
+            return None
+
+    @staticmethod
+    def resource_value(kind, cwl_value):
+        if kind == 'cpu':
+            return '{}'.format(cwl_value)
+        elif kind == 'memory':
+            return '{}Mi'.format(cwl_value)
```

Comment:
I expected this `resource_value` to `else: return None` to match the two above functions. I think python defaults to returning None anyway, so this is pretty pedantic.

Reply:
Good catch. That was an oversight.